test/fixtures/catalog/156F64.yaml

```diff
@@ -0,0 +1,25 @@
+id: 156F64
```

Comment:
I wouldn't have the same real check that we have in the catalog here. In fact, I would try not to add a new test fixture, as we are trying to remove them for the evaluation test cases in other PR. It will be possible to use factories to create this use cases